### ISSUE TITLE
Implement Wizard Stage 6 (Terms)

### ISSUE THREAD

**Issue by dkent600** [PrimeDAO/prime-deals-dapp#170]

**Comment by BartuCulha** [comment]
![Create a Partnered Deal _ Prime Deals](https://user-images.githubusercontent.com/24368665/155545845-9ca46462-68d3-4acb-99be-48ec151ee398.png)

In prime launch the placeholder text is removed, which looks fine. So my suggestion is to either remove it or change the placeholder text to:

"Please enter a clause"

Any opinions? @blink-km

**Comment by BartuCulha** [comment]
**Delete button stuck at focused state**

Steps to reproduce:
Navigate to: https://prime-deals-staging.vercel.app/initiate/token-swap/partnered-deal/terms
Click delete button inside Clause 1
![Create a Partnered Deal _ Prime Deals (1)](https://user-images.githubusercontent.com/24368665/155546985-57225fb0-a228-4a6e-9a02-cda1da42b666.png)

- [ ] Delete Button should go back to default state after clicked

**Comment by BartuCulha** [comment]
**Add A red border when we are throwing an input field error**

Steps to reproduce:
Navigate here: https://prime-deals-staging.vercel.app/initiate/token-swap/partnered-deal/terms
Click Add Clause
Click Proceed

You'll notice that first input field has a red border but second input field will not.
![Create a Partnered Deal _ Prime Deals (2)](https://user-images.githubusercontent.com/24368665/155547872-44f72ff8-49bd-41e5-8fd2-3013a0f887f7.png)

- [ ] Red border is added to all invalid input fields

**Comment by hiaux0** [comment]
#### Delete button should reset validation (only 1 clause present)
1. Have 1 clause only
2. Click on Delete
3. Validation visuals should disappear

@BartuCulha thoughts?
👍 if you agree 
👎 if don't

![grafik](https://user-images.githubusercontent.com/30693990/157015571-43a22b80-034a-41ce-8d78-a69f31edfcf8.png)

**Comment by curvelabs-dev** [comment]
> #### Delete button should reset validation (only 1 clause present)
> 1. Have 1 clause only
> 2. Click on Delete
> 3. Validation visuals should disappear
> 
> @BartuCulha thoughts? 👍 if you agree 👎 if don't
> 
> ![grafik](https://user-images.githubusercontent.com/30693990/157015571-43a22b80-034a-41ce-8d78-a69f31edfcf8.png)

I disagree.  I think that the Delete button should not be visible when there is only one clause.  Same for Tokens in Token Details.  I will create a ticket for this.
